express-api/tests/testUtils/factories.ts

```diff
@@ -231,3 +232,54 @@ export const produceEmail = (props: Partial<IEmail>): IEmail => {
   };
   return email;
 };
+
+export const produceBuilding = (): Building => {
```

Comment:
It would be better to have more of these values filled in for future use of this function in other tests, but probably sufficient for now.